int/docker-compose.yml

```diff
@@ -35,6 +35,7 @@ services:
 
   tests:
     container_name: juggernaut
+    command: -k test_holder_credential_exchange
```

Comment:
Let's drop this line so it runs everything by default!

Reply:
Thanks for catching that!